packages/geo/src/lib/print/shared/print.service.ts

```diff
@@ -115,6 +169,31 @@ export class PrintService {
 
     return status$;
   }
+  // ref GeoMoose https://github.com/geomoose/gm3/tree/main/src/gm3/components/print
+  addGeoRef(doc, map, width, height, resolution, margins) {
+    const unit = 'mm';
+    const docHeight = doc.internal.pageSize.getHeight();
+
+    // x,y = margin left-bottom corner for img in pdf doc
+    const x = margins[3];
+    const y = docHeight - margins[0] - height;
+
+    let pdf_extents = [x, y, x + width, y + height];
+    for(let i = 0; i < pdf_extents.length; i++) {
+      pdf_extents[i] = this.pdf_units2points(pdf_extents[i], unit);
+    }
+
+    // exemple in geoMoose
+    // let map_extents = map.viewController.olView.calculateExtent([
+    //   this.toPoints(width, unit) * resolution,
+    //   this.toPoints(height, unit) * resolution,
+    // ]);
+
+    const mapSize = map.ol.getSize();
+    const mapExtent = map.ol.getView().calculateExtent(mapSize);
+    doc.setGeoArea(pdf_extents, mapExtent);
```

Comment:
validate if mapextent are in 3857.

Reply:
yes ok! so if the map isn't in 3857 we do not georef ... the plugin is config for 3857.

Reply:
the georef will be written to the geopdf.. but will be wrong.

Reply:
take a look to https://openlayers.org/en/latest/apidoc/module-ol_proj.html#.transformExtent

Reply:
yes, in the plugin it work for 3857, so i will check and if isn't 3857, not reference that pdf.

Reply:
fix now getExtent is made in 3857. Thanks